### PR TITLE
Add ep mypad into list of supported plugins

### LINKED ISSUE #59: Add management for the plugins `ep_mypads`
## Affected Puppet, Ruby, OS and module versions/distributions

- Distribution: All supported nodesets
- Module version: 2.1.0

## What are you seeing

Add the possibility to install `ep_mypads` as a supported complex plugin like `ep_button_link`.

### PR DESCRIPTION
**Pull request by your1p**
#### Pull Request (PR) description

Add the possibility to install ep_mypads as a supported complex plugin like ep_button_link.

#### This Pull Request (PR) fixes the following issues

Fixes #59

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,4 @@ Puppetfile.lock
 .*.sw?
 .yardoc/
 Guardfile
+.rake_tasks~
```
Comment (bastelfreak) [inline .gitignore]: where is this file coming from? Our modulesync  config wil purge it. The template is https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.gitignore.erb

File: manifests/init.pp [hunk 2]
```diff
@@ -48,7 +49,9 @@
   String $default_pad_text          = 'Welcome to etherpad!',
 
   # Users
-  Optional[Hash] $users = undef,
+  Optional[Hash] $users                  = undef,
+  Optional[String] $ep_local_admin_login = 'admin',
```
Comment (bastelfreak) [inline manifests/init.pp]: can you use ```String[1]``` here and for the other new param?

File: .gitignore (hunk 1 above)
Comment (your1p) [reply .gitignore]: Fixed

File: manifests/init.pp (hunk 2 above)
Comment (your1p) [reply manifests/init.pp]: Fixed.

**Comment by your1p** [comment]
@llowder No, it can be an 'alert' because default LDAP parameters are example but any errors will be created. The user can't connect to any LDAP (because he don't configure anyone) but the installation behavior is the same. 
If you think it's better, I can change the message of the alert.

File: manifests/plugins/ep_mypads.pp [hunk 3]
```diff
@@ -0,0 +1,30 @@
+class etherpad::plugins::ep_mypads {
+  etherpad::plugins::common { 'ep_mypads' :
```
Comment (bastelfreak) [inline manifests/plugins/ep_mypads.pp]: this is a private class, right? Please add [assert_private()](https://forge.puppet.com/puppetlabs/stdlib#assert_private)

Comment (your1p) [reply manifests/plugins/ep_mypads.pp]: Yes, is a private class. I have adding `assert_type()`

**Comment by bastelfreak** [comment]
Thanks!

**Comment by your1p** [comment]
You are welcome. Thank you to!

**Pull request merged**